frontend/src/old-pages/Clusters/Instances.tsx

```diff
@@ -140,12 +157,6 @@ export default function ClusterInstances() {
     [],
   )
 
-  const clusterPath = ['clusters', 'index', clusterName]
-  const fleetStatus: ComputeFleetStatus = useState([
-    ...clusterPath,
-    'computeFleetStatus',
-  ])
-
   const onSelectionChangeCallback = React.useCallback(({detail}) => {
```

Comment:
issue: not related to this PR, but we noticed that since `instances` is continuously fetched from the backend, it's content changes, but the `selectedInstances` keeps a reference to the old/stale value

for instance, if the istance in the `instances` array goes from status=Running to status=Stopped, this change is never reflected in the `selectedInstance`

Reply:
Thanks, will track this issue in a separate task.